scipy/stats/_entropy.py

```diff
@@ -123,9 +123,9 @@ def entropy(pk: np.typing.ArrayLike,
 
     >>> D = entropy(pk, qk, base=base)
     >>> D
-    0.7369655941662062
-    >>> D == np.sum(pk * np.log(pk/qk)) / np.log(base)
-    True
+    0.736965594166206
+    >>> np.sum(pk * np.log(pk/qk)) / np.log(base)
```

Comment:
How about using the actual value that you're getting for `D to avoid the "slight lie" and then using `np.allclose` with a very small `rtol` (around twice machine epsilon?).

Reply:
See 3da2b2d.